documentation/source/building.rst

```diff
@@ -301,6 +301,15 @@ Makefile-based Test Scripts
     You can usually write out such a file from the simulator's GUI.
     This is currently supported for the Mentor Questa, Mentor ModelSim and Aldec Riviera simulators.
 
+.. make:var:: TOPLEVEL_LANG
+
+    Used to inform the makefile scripts which HDL language the top-level design element is written in.
+    Currently it supports the values ``verilog`` for Verilog or SystemVerilog tops, and ``vhdl`` for VHDL tops.
+    This is used by simulators that support more than one interface (VPI, VHPI, or FLI) to select the appropriate interface to start cocotb.
+
+    The variable is also made available to cocotb tests conveniently as :data:`cocotb.LANGUAGE`.
+    This can be used to prevent individual tests from running with the selected :make:var:`TOPLEVEL_LANG`.
```

Comment:
Perhaps move this sentence to the docs for `cocotb.LANGUAGE`, maybe as:
```
.. info::
    cocotb's own regression tests sometimes run the same tests against both a
    Verilog and VHDL implementation of the same module, where this variable
    can be used to skip tests that only make sense in one of the two languages.
```

Reply:
I removed the sentence, but I'm hesitant to document the use of `LANGUAGE` because I'm not sure if we should be doing that *at all*. Really tests should run agnostic to the implementation language, that's one of the main points of the GPI. If this is necessary we should be segregating the tests into different suite where in the makefiles we decide whether to skip the whole suite or not.